cpp/src/neighbors/mg/mg.cuh

```diff
@@ -17,6 +17,7 @@
 #pragma once
 
 #include "../detail/knn_merge_parts.cuh"
+#include <raft/comms/nccl_clique.hpp>
```

Comment:
Seeing the following build failure [in the ARM CUDA 11.8 wheel CI job]( https://github.com/rapidsai/cuvs/actions/runs/14606972256/job/40977947859?pr=835#step:11:551 ):

```cpp
  In file included from /__w/cuvs/cuvs/cpp/src/neighbors/mg/mg_pq_half_int64_t.cu:26:
  /__w/cuvs/cuvs/cpp/src/neighbors/mg/mg.cuh:20:10: fatal error: raft/comms/nccl_clique.hpp: No such file or directory
     20 | #include <raft/comms/nccl_clique.hpp>
        |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Does this header get packaged in our wheels?

Reply:
sorry - I misread the raft change, and that file has been removed entirely in https://github.com/rapidsai/raft/pull/2549 (I thought we just were relying on a transitive include inside of raft and that the include had been removed, not that the entire file had been removed).

Changing cuvs to match raft is a bigger change than this unfortunately =(